app/workers/push_update_worker.rb

```diff
@@ -8,7 +8,9 @@ def perform(account_id, status_id)
     status  = Status.find(status_id)
     message = InlineRenderer.render(status, account, 'api/v1/statuses/show')
 
-    Redis.current.publish("timeline:#{account.id}", Oj.dump(event: :update, payload: message, queued_at: (Time.now.to_f * 1000.0).to_i))
+    prefix = ENV.fetch('REDIS_PUBSUB_PREFIX') { '' }
```

Comment:
Since this code is repeated a few times, what do you think about putting it either in a helper function or as a frozen constant somewhere?